### PR TITLE
add ovirt_vnic_profile_info

### PR DESCRIPTION
**Pull request by mnecas**
##### SUMMARY

add ovirt_vnic_profile_info

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1746846
##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

ovirt
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/ovirt/ovirt_vnic_profile_info.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_vnic_profile_info
+short_description: Retrieve information about one or more oVirt/RHV vnic profiles
+author: "Martin Necas (@mnecas)"
+version_added: "2.9"
```
Comment (machacekondra) [inline lib/ansible/modules/cloud/ovirt/ovirt_vnic_profile_info.py]: 2.10

File: lib/ansible/modules/cloud/ovirt/ovirt_vnic_profile_info.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        module.exit_json(changed=False, **result)
+    except Exception as e:
+        module.fail_json(msg=str(e), exception=traceback.format_exc())
+    finally:
+        connection.close(logout=auth.get('token') is None)
+
+
+if __name__ == '__main__':
+    main()
+
```
Comment (machacekondra) [inline lib/ansible/modules/cloud/ovirt/ovirt_vnic_profile_info.py]: new line

File: lib/ansible/modules/cloud/ovirt/ovirt_vnic_profile_info.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+author: "Martin Necas (@mnecas)"
+version_added: "2.9"
+description:
+    - "Retrieve information about one or more oVirt/RHV vnic profiles."
+notes:
+    - "This module returns a variable C(ovirt_vnic_profiles), which
+       contains a list of vnic profiles. You need to register the result with
+       the I(register) keyword to use it."
+options:
+    max:
```
Comment (machacekondra) [inline lib/ansible/modules/cloud/ovirt/ovirt_vnic_profile_info.py:43]: can you add also name parameter?

**Review by machacekondra — APPROVED** [review]
shipit

**Review by mwperina — APPROVED** [review]
shipit

**Comment by machacekondra** [comment]
@Akasurde Can you please review/merge?

**Pull request merged**